src/main/java/org/fcrepo/migration/foxml/ArchiveExportedFoxmlDirectoryObjectSource.java

```diff
@@ -56,8 +64,17 @@ public void setFetcher(final URLFetcher fetcher) {
         this.fetcher = fetcher;
     }
 
+    /**
+     * Sets a FileFilter to determine which files will be considered as object
+     * files in the source directories.
+     * @param fileFilter a FileFilter implementation
+     */
+    public void setFileFilter(final FileFilter fileFilter) {
```

Comment:
When a user wishes to override the inclusion pattern, that individual will be able to provide an alternate FileFilter as a property to this bean in the spring xml configuration.  I considered adding an example to all of the configuration files, but opted against it because the goal is to keep those files simple.  If in the future anyone runs across files they wish to ignore that don't start with '.' there will be a way that doesn't require a code change.